src/utils/time/index.js

```diff
@@ -20,24 +20,24 @@ export const calculateTime = (time, formatForChatTab = false) => {
   }
 
   if (minutes < 60) {
-    return minutes === 0 ? '1m ago' : `${minutes}m ago`;
+    return minutes === 0 ? '1m' : `${minutes}m`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code removes the 'ago' suffix from the time representation. This might lead to confusion for users as they may not understand if the displayed time is a duration or a specific point in time. Consider keeping the 'ago' suffix for clarity.
```diff
- return minutes === 0 ? '1m' : `${minutes}m`;
+ return minutes === 0 ? '1m ago' : `${minutes}m ago`;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->